DEPLOY.md

```diff
@@ -0,0 +1,683 @@
+
+With this feature walkthrough, we will be simulating the automatic process recovery by terminating one of the service within Lagom Chirper.
+
+Scale `Activity Stream` to `2` instances.
+
+```
+conduct run activity-stream-impl --scale 2
+```
+
+Once scaled, the overall state should be similar to this. The `activity-stream-impl` which is the bundle name of the `Activity Stream` service now has `2` running instances.
+
+```
+$ conduct info
+ID               NAME                  VER  #REP  #STR  #RUN  ROLES
+89fe6ec          activity-stream-impl   v1     1     0     2  web
+73595ec          visualizer             v2     1     0     1  web
+bdfa43d-e5f3504  conductr-haproxy       v2     1     0     1  haproxy
+6ac8c39          load-test-impl         v1     1     0     1  web
+9a2acf1-44e4d55  front-end              v1     1     0     1  web
+3349b6b          eslite                 v1     1     0     1  elasticsearch
+01dd0af          friend-impl            v1     1     0     1  web
+d842342          chirp-impl             v1     1     0     1  web
+1acac1d          cassandra              v3     1     0     1  cassandra
+```
+
+We should be able to see `2` process ids belonging to `Activity Stream` service. Take note of the value of both process ids.
+
+```bash
+$ pgrep -f activity-stream-impl
+```
+
+Let's kill one of the processes.
+
+```bash
+$ pgrep -f activity-stream-impl | head -n 1 | xargs kill
+```
+
+Eventually we should still see `2` process ids belonging to `Activity Stream` service, with a new process id replacing the one we killed. The new process is being started quite quickly, and it's very likely you will see the new process id when you run this command.
+
+```bash
+$ pgrep -f activity-stream-impl
+```
+
+You may choose to scale `Activity Stream` back down to `1` instance if you wish.
+
+```bash
+conduct run activity-stream-impl --scale 1
+```
+
+#### Rolling upgrade of the app
+
+Performing rolling upgrade on [Production Suite](https://www.lightbend.com/platform/production) is relatively straightforward.
+
+New version of a service can be deployed and run alongside an existing version. Should the new version exposes the same endpoint as the existing, while running alongside each other the traffic from the proxy will be delivered to both new and existing version in round-robin fashion.
+
+In this feature tour, we will be performing a rolling upgrade of the `Friend` service.
+
+There is an instance of `Friend` service already running - look for the bundle with `friend-impl` as its `NAME`. Note the `ID` of the `friend-impl` service - it has `3ff82bf` as its value. We call this identifier "Bundle Id".
```

Comment:
I think `3ff82bf` is used throughout the docs and is a leftover from previous runs. In this case in particular it's mentioned in the paragraph but not listed in the following output sample. If that is correct and iintentional I find it a bit confusing.